std_msgs/Header header
uint16[] xxyy
int16 width
int16 height